lib/src/models/user.dart

```diff
@@ -5,10 +5,12 @@ class User {
   final String firstName;
   final String lastName;
   final List<Profile> profiles;
+  final bool isGuest;
 
   User.fromJson(Map<String, dynamic> json)
       : id = json['id'],
         firstName = json['firstName'],
         lastName = json['lastName'] ?? '',
+        isGuest = json['guest'] ?? true, // This information comes from table `user_property_value` and defaults to true in platform (see SubscriptionManager.isGuestSafe(userId))
```

Comment:
I'm not a fan of this and hope we can eliminate it in a future refactor. If we're trying to determine whether a user has a plan/subscription, we can use the usage data for that.

Reply:
I didn't know that this information could be found in usage... Is this the `lastName` that you are referring to? If this is the case, this is not super pretty. Also, why don't you like this information with the User information. This, to me, seems to go well together.